.github/workflows/rust.yml

```diff
@@ -89,6 +88,17 @@ jobs:
         run: brew install libtool
         if: runner.os == 'macOS'
 
+      # We cache protoc because it sometimes fails to download, but cache is too slow on Windows.
+      - name: Configure tool cache
+        uses: actions/cache@1bd1e32a3bdc45362d1e726936510720a7c30a57 # v4.2.0
+        id: tool-cache
+        with:
+          path: '~/**/_tool'
```

Comment:
What about specifying a full path rather than `**/_tool` glob?

Reply:
The paths are different on macOS and Linux, so this glob is the easiest way to write them both.

Reply:
I see, annoying :confused: